stage0/src/Lean/Data/Rat.lean

```diff
@@ -0,0 +1,125 @@
+/-
```

Comment:
this appears to be a new file about rational numbers, what does that have to do with this PR ?

Reply:
Please ignore all changes in `stage0/` :) . I couldn't even find your comment at first in order to respond because they are all collapsed by default.